PowerQueryM/README.md

```diff
@@ -0,0 +1,16 @@
+This folder has three sample Power Query M queries that can be used to load MCast™ data into Excel or PowerBI.
+
+The first sample, ObservedLoad.query.pq, uses the MCast™ API to populate a table with the last 60 days of consumption data. The second sample, ForecastedLoad.query.pq, uses the MCast™ API to populate a table with all of the pinned one-day-ahead MCast™ forecast values from the last 60 days. The third sample, APE.query.pq, merges the results of the previous two queries and adds columns for the day of week and the absolute percent error.
+
+To run these samples, open Excel and in the "Data" tab, select "Get Data" -> "From Other Sources" -> "Blank Query" to create a new query. 
+
+![BlankQuery]
+
+In your new query, under "Query," select "Advanced Editor" and paste in the code from any of the three samples. For ObservedLoad.query.pq or ForecastedLoad.query.pq, you'll have to edit the query to use your own domain and API Key. For APE.query.pq, you'll have to setup the other two examples first.
+
+![AdvEditor]
+
+Finally, select "Close & Load" to finish setting up the query. By default, any connection will load in a tab in a new table. 
+
+[BlankQuery]: ../assets/BlankQuery.png "Create Blank Query"
```

Comment:
Hmm I was kind of expecting to see it lead you to "form Web" tab since that seems to be set up specifically for API calls. Have you looked into this at all?

Reply:
That's how I started when I created this sample. But that's what you do to build a web query from scratch. I could see something in our help docs explaining it, but if you have an already built query, the only way I know to load it is via a blank query and copy&paste

Reply:
Yeah. I can just see the even copy pasting files looking scary to someone. But I could also see that example being something in the help docs and then linking to this github then if you want to automate that pull?

Reply:
Yeah, I mean _walking through_ this sort of analysis without providing code to run would be a little too much I think. I'm in favor of something in the help docs that just guides you to a very simple no-code pull of the data, and then this code sample being available to do an actual analysis.
But like we had a user who was doing a no-code solution and couldn't figure out how to pull multiple days (with the whole `Date.AddDays` part), so I think there's got to be some sample code for folks to reference.

Reply:
Yeah I like this as the solution. I like the idea of a Loom video (@jdrenovsky) on our help docs for the completely no-code solution. I think it should then end with pulling up the advanced editor so you can see what code was auto-generated by the process and leave it at if you want to further automate this process you can make edits here and go to our github page for a more in-depth analysis